ext/standard/tests/strings/gh10940.phpt

```diff
@@ -0,0 +1,9 @@
+--TEST--
+Test unpacking at the 32-bit integer limit
+--FILE--
+<?php
+$a = pack("AAAAAAAAAAAA", 1,2,3,4,5,6,7,8,9,10,11,12);
+unpack('h2147483647', $a);
+?>
+--EXPECTF--
+Warning: unpack(): Type h: not enough input, need 1073741824, have 12 in %s on line %d
```

Comment:
Jesus this warning is incomprehensible, maybe something we should fix in master

Reply:
Something like this maybe: "Type h: not enough input elements, need X elements but only Y were passed in %s on line %d" ?

Reply:
Yeah that read better already